torch/csrc/jit/passes/quantization.cpp

```diff
@@ -101,20 +101,17 @@ void PropagateQuantInfo(std::shared_ptr<Graph>& graph) {
   throw std::runtime_error("Pass not implemented yet!");
 }
 
-static void addObserverFor(Value* v, Node* original_observer_node) {
-  Node* def = v->node();
+static Node* addObserverFor(Value* v, Node* original_observer_node, Node* def) {
+  AT_ASSERT(def != nullptr);
   WithInsertPoint ins(def);
```

Comment:
`ins` is not used?

Reply:
`WithInsertPoint` sets an insertion point in constructor and restores the previous insertion point in the destructor. Thus, `ins` is never used directly, but it is still useful :)